sample/benchmark/src/main/java/com/datadog/benchmark/sample/MainActivity.kt

```diff
@@ -38,6 +39,7 @@ class MainActivity : AppCompatActivity() {
         datadogBenchmark.stop()
     }
 
+    @OptIn(ExperimentalRumApi::class)
```

Comment:
It seems it is not needed here? I don't see any `addViewLoadingTime` call here

Reply:
yup a left - over from previous tries :(